doc/gh-pages/notation/filters.md

```diff
@@ -7,9 +7,32 @@ nav_order: 11
 # Conditions & Filters
 
 {: .story }
->TODO
+>**GM**: You are lucky. The potion saved your life, cured all your wounds and gaved you an extra boost of energy. As you get up, you realize that you are in a room filled with bags of golden coins.
+>
+>**Grindon**: The heart of the dunjeon!..
+>
+>**GM**: Yes. Before anyone appear and catch you, you grab as many bags you can in your arms.
+>
+>**Grindon**: Finally, all this time training in the woods will serve me well. I surely can hold 8 of those nice bags. Let's roll 8d20.
+>
+>**GM**: True but hold on. The bags are slippery. And small bags containing 4 coins or less may fall from your arms.
+>
+>**Grindon**: Okay, let's roll. Please dice, be kind.
+>
+>*GNOLL*: [{16, 1, 5, 8, 1, 9, 1, 13}]
+>
+>**GM**: You escape the dunjeon with...
+>
+>**Grindon The Brave**: Not so fast, dear GM. I see lots of 1s here. It is time for my special class feature. I knew it would come handy.
```

Comment:
```suggestion
>**Grindon The Brave**: Not so fast, dear GM. This is my time to shine. I have a special class feature that lets me reroll dice when it's about losing money. I knew it would come handy! 5d20>4!  

```

Reply:
Yes, disregard the dice suggestion. I made a mistake as I mentioned in another comment